src/components/event/EventCardHeader.tsx

```diff
@@ -82,9 +83,23 @@ function EventCardHeader({
 		e.stopPropagation();
 	};
 
+	const hoverEvent = useDebouncedCallback(() => {
```

Comment:
This way hoverEvent could be triggered even when card isn't hovered.

create ref with timeout. onMouseOver assign ref a timeout with delay. onMouseLeave clear timeout.
I did something similar in TimeSelector component inside GraphChunksVirtualizer file